src/Tools/ExternalAccess/AspNetCore/Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.csproj

```
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.AspNetCore</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.AspNetCore</PackageId>
+    <PackageDescription>
+      A supporting package for ASP.NET Core:
+      https://github.com/dotnet/aspnetcore
+    </PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\..\Compilers\Core\Portable\InternalUtilities\IsExternalInit.cs" Link="Internal\Utilities\IsExternalInit.cs" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Remote.ServiceHub" />
+
+    <!--
+      ⚠ ONLY ASP.NET CORE ASSEMBLIES MAY BE ADDED HERE ⚠
+    -->
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.UnitTests" />
```

Consumer of this API will either be in `Microsoft.AspNetCore.Analyzers` or `Microsoft.AspNetCore.App.Analyzers` or a new assembly. Add the two known ones for now. Can lock this down once we know where code will end up.
```suggestion
    <InternalsVisibleTo Include="Microsoft.AspNetCore.Analyzers" />
    <InternalsVisibleTo Include="Microsoft.AspNetCore.App.Analyzers" />
    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.UnitTests" />
```